src/Persistence/Microsoft.PowerPlatform.PowerApps.Persistence.csproj

```diff
@@ -32,7 +32,8 @@
     <PublicSign>true</PublicSign>
     <!-- Workaround for version range https://github.com/NuGet/Home/issues/11842 -->
     <RestorePackagesWithLockFile>true</RestorePackagesWithLockFile>
-    <NoWarn>$(NoWarn);NU1601</NoWarn>
+    <!-- YamlValidator Supressions: CA1822, IDE0052 (unused json serializer) in YamlValidator -->
+    <NoWarn>$(NoWarn);NU1601;CA1822;IDE0052</NoWarn>
```

Comment:
undo these suppressions.
Instead, either address the diagnostics, or suppress them in one of the many ways. Often, I prefer using the attribute method for things. But you can also update the .editorconfig